lib/puppet/provider/user/hpux.rb

```diff
@@ -63,4 +75,43 @@ def password
       return ent.passwd
     end
   end
+
+  def trusted
+      trusted_sys = %x(/usr/lbin/getprpw root 2>&1)
+      if trusted_sys.chomp == "System is not trusted."
+         "NotTrusted"
+      else
+         "Trusted"
+      end
+  end
+
+  def trust2
```

Comment:
This appears to be nearly identical to #trusted, and unused. Why is it included?